CardData.py

```diff
@@ -50,15 +50,6 @@
 # Gaunter's 'Higher than 5' and 'Lower than 5' are not actually cards.
 INVALID_TOKENS = ['200175', '200176']
 
-"""
- Use the ID from here to look up localisations in categories output file.
- WARNING : this list is not simply 1 to 73, some numbers are missing
-"""
-CATEGORIES = [1, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 2, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 3, 30, 
```

Comment:
You removed CATEGORIES ? So for some cards the JSON will show a category that doesn't exist in the translations. Isn't that a problem ?

Reply:
Are there cards that do that?

Reply:
I didn't like the fact that we were reliant on a known list of Categories, it opens us up to mistakes (e.g. new categories get introduced to the game and we don't notice). I much prefer this automated way of doing it.

We could build the list of known categories from the translations files, but CDPR sometimes do their translations in stages. For example sometimes they have the english and polish translations in the game files, but won't have the other languages ready until the patch is released. There is a potential scenario where we build the list of categories from the English translations, but the category exists in all languages other than English. In this case we wouldn't include it the JSON even though the category exists.

I think it is reasonable to give the responsibility of making this check to the Developers. They can then decide how to handle it in their project.

Open to discussion though. @Zolmeister @Tri125 thoughts?

Reply:
@Zolmeister 

Kaedweni Sergeant. The bit mask resolves to 3 categories, one of them is 45. Category 45 doesn't exist in the translation files.

Reply:
If cards can reference incomplete/nonexistant categories, then we should reference the categories directly and only include those which have translations for every locale.

Reply:
What happens if someone has a project that only uses one language? It feels unfair to exclude categories from them just because they don't exist in a language that they don't use

Reply:
ok, then keep all the categories from the card and simply document that categories may not exist or have all translations.